requirements/base.txt

```diff
@@ -80,23 +82,25 @@ flask-appbuilder==4.1.6
     # via apache-superset
 flask-babel==1.0.0
     # via flask-appbuilder
-flask-caching==1.10.1
+flask-caching==2.0.1
     # via apache-superset
-flask-compress==1.10.1
+flask-compress==1.13
     # via apache-superset
 flask-jwt-extended==4.3.1
     # via flask-appbuilder
-flask-login==0.4.1
-    # via flask-appbuilder
-flask-migrate==3.1.0
+flask-login==0.6.2
```

Comment:
this is a major bump, isn't out of scope given the description?

Reply:
@dpgaspar I had a error with the new flask and old `flask-login` and I bumped it